src/driver/driver.py

```diff
@@ -21,12 +21,12 @@ def driver_loop():
 
 def add_gps_coord(route: List[GpsCoord], coord: GpsCoord) -> List[GpsCoord]:
     """Adds a GpsCoord to the route"""
-    pass
+    return route.append(coord)
 
 
 def remove_gps_coord(route: List[GpsCoord], coord: GpsCoord) -> List[GpsCoord]:
     """Removes a GpsCoord to the route"""
-    pass
+    return route.remove(coord)
```

Comment:
.remove also returns `none`. We can actually do this with list comprehension to return a new list:
https://www.w3schools.com/python/python_lists_comprehension.asp